base/applications/clipbrd/clipbrd.c

```diff
@@ -270,12 +303,15 @@ static void OnPaint(HWND hWnd, WPARAM wParam, LPARAM lParam)
     PAINTSTRUCT ps;
     COLORREF crOldBkColor, crOldTextColor;
     RECT rc;
+    HGDIOBJ hFontOld;
 
     if (!OpenClipboard(Globals.hMainWnd))
         return;
 
     hdc = BeginPaint(hWnd, &ps);
 
+    hFontOld = SelectObject(hdc, Globals.hFont);
```

Comment:
Original code was able to correctly display text without having to specify a font there. Why do you need that now?

Reply:
The window default font is not monospace'ed nand maybe ugly in some environments.

Reply:
Deleted `hFont` member.